Filter/PagerfantaFilter.php

```diff
@@ -3,37 +3,37 @@
 namespace LAG\AdminBundle\Filter;
 
 use Symfony\Component\HttpFoundation\Request;
-use Symfony\Component\HttpFoundation\ParameterBag;
 
-class PagerfantaFilter
+class PagerfantaFilter extends RequestFilter
```

Comment:
Maybe only the adapter should hold the logic of how PagerFanta is filtering the request.